mne/io/persyst/tests/test_persyst.py

```diff
@@ -16,10 +17,10 @@
 from mne.utils import run_tests_if_main
 
 fname_lay = op.join(
-    data_path(download=False), 'Persyst',
+    data_path(download=True), 'Persyst',
```

Comment:
Should not need to change this and it will break things if you do, please revert